charts/nextcloud/templates/deployment.yaml

```diff
@@ -371,8 +409,27 @@ spec:
       # Will mount configuration files as www-data (id: 82) for nextcloud
       securityContext:
         fsGroup: 82
+        {{- if .Values.securityContext }}
+        {{- with .Values.securityContext }}
+        {{- toYaml . | nindent 10 }}
```

Comment:
Indentation of 10 is wrong should probably be 8.

Reply:
fixed